src/CoreBundle/Bridge/Chameleon/Module/NavigationTree/NavigationTree.php

```diff
@@ -572,7 +572,12 @@ private function createRootTree(): array
             $defaultPortal = $this->portalDomainService->getDefaultPortal();
             $defaultPortalMainNodeId = $defaultPortal->fieldMainNodeTree;
 
-            $portalList = \TdbCmsPortalList::GetList();
+            $portalsInMainTreeNodeOrderQuery = '
+                SELECT `cms_portal`.* FROM `cms_portal` 
+                    LEFT JOIN `cms_tree` ON `cms_portal`.`main_node_tree` = `cms_tree`.`id` 
+                ORDER BY `cms_tree`.`lft`, `cms_portal`.`sort_order`
```

Comment:
Why order by lft?

Reply:
The nested set is used to render all the other parts of the navigation tree. In my opinion, it makes sense to use it for the portal-level too.